### ISSUE TITLE
Altitude empty every other sample

### ISSUE THREAD

**Issue by awpeters** [balta3/sailfish-gpsinfo#10]
As the title says: altitude reading alternates between a number and a dash (not available?)

**Comment by balta3** [comment]
Yes, the dash means not a valid value. SailfishOS is giving me an invalid flag every second sample, I do not know if this is a bug in SailfishOS or the Qt-wrapper for the hardware. I already thought about only showing the dash if at least two successive invalid flags are received.

I will keep you updated.
